### ISSUE TITLE
Added Movie Doesn't Appear in Movies List and Gives 404

### ISSUE THREAD

**Issue by Hossy** [Radarr/Radarr#2672]
**Added Movie Doesn't Appear in Movies List and Gives 404**

I'm running into an interesting problem trying to add a movie.  The movie is The Wedding Planner (https://www.themoviedb.org/movie/2018-the-wedding-planner).  When I search to add it, radarr tells me it already exists.  However, it does not appear in my movie list.  Also, when I click the Already Exists button (which normally takes you to the movie in radarr), I get the green 404 monster.  Any idea on how to fix this?  Sounds like data corruption.

Provide a description of the feature request or bug here, the more details the better. 
Please also include the following if you are reporting a bug. If you do not include it, the issue will probably be closed as we cannot help you.

**Radarr Version:** 0.2.0.995

**Mono Version:** 5.10.1.4

**Trace Logs:**

The trace logs only show my search so I could click the Already Exists button.  No logs from the 404 after clicking the button.

18-3-24 18:16:13.7|Trace|Http|Req: 2867 [GET] /api/movie/lookup?term=the+wedding+planner
18-3-24 18:16:13.7|Debug|Parser|Parsing string 'the wedding planner'
18-3-24 18:16:13.7|Debug|Parser|Unable to parse the wedding planner
18-3-24 18:16:13.7|Trace|HttpClient|Req: [GET] https://api.themoviedb.org/3/search/movie?api_key=**********&query=the%2Bwedding%2Bplanner&year=&include_adult=False
18-3-24 18:16:14.1|Trace|HttpClient|Res: [GET] https://api.themoviedb.org/3/search/movie?api_key=**********&query=the%2Bwedding%2Bplanner&year=&include_adult=False: 200.OK (342 ms)
18-3-24 18:16:14.1|Trace|Http|Res: 2867 [GET] /api/movie/lookup?term=the+wedding+planner: 200.OK (347 ms)
18-3-24 18:16:14.1|Debug|Api|[GET] /api/movie/lookup?term=the+wedding+planner: 200.OK (347 ms)

**Comment by Hossy** [comment]
Digging into this a little more, I was able to fix the problem and I determined WHY it happened.  However, I'm concerned this can easily happen again, so I'm thinking this is a bug request against the Already Exists button (allow me to explain)...

https://www.themoviedb.org/movie/2018-the-wedding-planner/changes

I added this movie prior on 3/19.  On 3/19, the original_title (in TMDB) for this movie was changed to "Ο Γάμος του Εραστή μου."  On 3/20, the original_title (in TMDB) for this movie was changed back to "The Wedding Planner."

Obviously, searching for "the wedding planner" in the search bar in radarr wasn't going to return a match because radarr uses the original_title when adding to the database.  However, when I searched via Add Movies for "the wedding planner", the movie appeared and showed me the Already Exists button.  So far, we're good.  This is desired behavior.  But here's the bug: the Already Exists button's hyperlink is created from data return from TMDB instead of being created from data from the radarr database.  So, when the original_title changes, the movie stub radarr tried to use changes.
